config/connect-log4j.properties

```diff
@@ -13,19 +13,27 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-log4j.rootLogger=INFO, stdout
+log4j.rootLogger=INFO, stdout, connectAppender
 
 
 log4j.appender.stdout=org.apache.log4j.ConsoleAppender
 log4j.appender.stdout.layout=org.apache.log4j.PatternLayout
 
```

Comment:
While many developers may be able to understand this section, many users will not immediately understand the behavior and purpose of these options. How about adding a comment that explains this new section, something like:

```
# Send the logs to a file, rolling the file at midnight local time. For example, the `File` option specifies the 
# location of the log files (e.g., `${kafka.logs.dir}/connect.log`), and at midnight local time the file is closed
# and copied in the same directory but with a filename that ends in the `DatePattern` option.
#
```